rcl/test/rcl/test_subscription.cpp

```diff
@@ -571,6 +571,10 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   {
     ret = rcl_subscription_fini(&subscription, this->node_ptr);
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+
+    test_msgs__msg__Strings__fini(&msg);
+    ret = rmw_serialized_message_fini(&serialized_msg);
+    ASSERT_EQ(RMW_RET_OK, ret) << rcl_get_error_string().str;
```

Comment:
I'll suggest combining these two lines for one particular reason.  The types here are all messed up.  That is, as far as I can tell, `rmw_serialized_message_fini` is a macro that just calls `rcutils_uint8_array_fini`.  Thus, the fact that it is documented as returning `RMW_*` error codes is a lie; it actually returns `RCUTILS_*` error codes.  This all ends up working because they are all integers, but this should really all be sorted out.  You don't need to do that in this patch, but I don't want to make the situation worse by assigning an `RCUTILS_*` error code to an `rcl_ret_t`.  Let's just do:

```
ASSERT_EQ(RMW_RET_OK, rmw_serialized_message_fini(&serialized_msg)) << rcl_get_error_string().str;
```

And leave the other cleanup for another time.

Reply:
Thanks for your comments.  
Got it.

Reply:
Thanks, appreciated.